Sources/DistributedActors/Cluster/SWIM/SWIMActorShell.swift

```diff
@@ -541,16 +1049,25 @@ internal struct SWIMActorShell {
         )
     }
 }
+*/
 
 extension SWIMActorShell {
     static let name: String = "swim"
     static let naming: _ActorNaming = .unique(SWIMActorShell.name)
 
     static var props: _Props {
-        _Props
-            ._wellKnown
-            .metrics(group: "swim.shell", measure: [.serialization, .deserialization])
+        var ps = _Props()
+        ps._knownActorName = ActorPath._swim.name
+//        ps._systemActor = true
```

Comment:
I guess not in the "new world" since system/not system really was about "where in the tree" they live, and since we don't have the tree in DA world, it doesn't matter as much. Ok to not have it as system.